src/wallet/db.cpp

```diff
@@ -126,6 +145,8 @@ void BerkeleyEnvironment::Close()
         if (database.m_db) {
             database.m_db->close(0);
             database.m_db.reset();
+            LOCK(cs_db);
+            g_fileids.erase(database.m_file_path);
```

Comment:
Do you have a specific scenario where this could happen? This change seems to be causing a test failure.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/18971#discussion_r440386812

> Do you have a specific scenario where this could happen? This change seems to be causing a test failure.

Marking resolved, followed up in a new thread: https://github.com/bitcoin/bitcoin/pull/18971#discussion_r441652829